base/server/python/pki/server/deployment/__init__.py

```diff
@@ -1385,22 +1385,6 @@ def import_master_config(self, subsystem):
                 'internaldb.ldapconn'
             ])
 
-        tags = subsystem.config['preop.cert.list'].split(',')
-        for tag in tags:
-            if tag == 'sslserver':
-                continue
-
-            # check CSR in CS.cfg
-            param = '%s.%s.certreq' % (subsystem.name, tag)
-            csr = subsystem.config.get(param)
-
-            if csr:
-                # CSR already exists
-                continue
-
-            # CSR doesn't exist, import from master
-            names.append(param)
-
         if subsystem.name == 'ca':
```

Comment:
I think this code needs to be restored, but instead of checking the `<subsystem>.<tag>.certreq` we should check the `<instance>/certs/<nickname>.csr` file. If the file exists we can skip it, but otherwise we need to import the CSR from the master.

Reply:
On the master side, we'll need to check if the requested param equals `<subsystem>.<tag>.certreq` we'll need to get the value from the CSR file:
https://github.com/dogtagpki/pki/blob/master/base/server/src/main/java/com/netscape/cms/servlet/csadmin/GetConfigEntries.java#L156

Reply:
I cannot verify what is present in the folder. It is not sure if it will be created but maybe I can check if they are provided. I'll do some tests.